project/gen2.scala

```diff
@@ -175,6 +175,26 @@ object gen2 {
         io.transact(xa).unsafePerformIO
       },
 
+      enum("GcalBaselineType") {
+        type GcalBaselineTypeRec = Record.`'tag -> String`.T
```

Comment:
I was using `ENUM` for types that have no structure other than being distinct from one another (i.e., when you would use `Symbol` in Scala). But it may be easier in the end to use lookup tables everywhere.